skipper/runner.py

```diff
@@ -124,9 +143,23 @@ def handle_volumes_bind_mount(docker_cmd, homedir, volumes, workspace):
     return docker_cmd
 
 
+# We have couple of special case mounts
+# 1. gitconfig file - it is required by skipper but may not exists, we don't want to create folder if it doesn't exist
+# that's why we create it as file
+# 2. .docker/config.json - if it is required and doesn't exists we want to create is as file with {} as data
+def _special_case_mounts(volume):
```

Comment:
Consider renaming (create_volume_local_path_if_not_exists)

Reply:
but it is relevant only for 2 cases

Reply:
i will move some logic there and rename